README.md

```diff
@@ -23,17 +23,54 @@
 # Swift-Kuery-ORM
 
 ## Summary
-Swift-Kuery-ORM is an Object-relational Mapping build on top of [Swift-Kuery](http://github.com/IBM-Swift/Swift-Kuery). Using Codable it provides an easy way of persisting your data.
+Swift-Kuery-ORM is an ORM (Object Relational Mapping) library built for Swift on the server. Using it allows you to simplify persistence of model objects with your server.
```

Comment:
drop "one the server" because it'll work on iOS as well (and is designed to be usable there - Cocoapod to come!)